### ISSUE TITLE
Search Settings changes don't stick

### ISSUE THREAD

**Issue by snowybunting** [pymedusa/Medusa#3705]
Master branch
CentOS 7.3

I make changes in the Search Settings/Torrent Search, changing from "Blackhole" to Transmission, I enter in all the transmission fields, hit the "Test Connection", that is successful. I save the changes, go to another tab, then go back to the Torrent Search, it has reverted back to Blackhole settings.

This is a working Transmission setup for SickRage, the difference is Medusa is on a CentOS box and transmission is on an Ubuntu box on the same local network. SickRage is also on the Ubuntu box, that setup works as expected.

I can pick a show to download, it will send the torrent to transmission and download it, but it won't process it any further - the status on the show page shows it as "Snatched".

Let me know what I can provide to further diagnose this.

**Comment by medariox** [comment]
> 
> **Before submitting your issue:**
> Enable debug logging in Medusa settings, reproduce the error (be sure to disable after the bug is fixed)

Please reopen once you have debug logs posted.

**Comment by snowybunting** [comment]
I wouldn't be posting the entire log, just the relevant parts?

**Comment by medariox** [comment]
Just the relevant parts.

**Comment by fernandog** [comment]
Isn't this easy to reproduce?

**Comment by fernandog** [comment]
I don't think debug logs will point an UI issue. Except error logs. And seems there is no one

**Comment by medariox** [comment]
I can't reproduce the issue, so I can't do anything without debug logs.

**Comment by snowybunting** [comment]
I'm not certain what *exactly* the issue was, but it seems to be connected to Firefox. When I connected to Medusa's webgui thru Chrome, the issue was not there.  I tried a couple of different browsers late last night, the only problem was on my default Firefox browser. I'll try to flesh it out later, but it seems to have something to do with the security settings in Firefox.

There was no sign of this in the logs.
